packages/fiber/src/index.tsx

```diff
@@ -17,3 +17,4 @@ export type { ObjectMap, Camera } from './core/utils'
 export * from './web/Canvas'
 export { createPointerEvents as events } from './web/events'
 export * from './core'
+export { Stage, FixedStage, Standard } from './core/stages'
```

Comment:
`Standard` feels like a very generic sounding constant to export from the package root.